SwiftSampleProject/SwiftSampleProject.xcodeproj/xcshareddata/xcschemes/SwiftyVokoder.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0800"
+   LastUpgradeVersion = "0820"
```

Comment:
Not sure if this even matters but any chance that this `LastUpgradeVersion` and `LastUpgradeCheck` above being different than the image in the travis.yml is messing up Travis all of a sudden?